### PR TITLE
Addressing further review by Med Moucadair

### PR DESCRIPTION
**Pull request by michael-scharf**

### CODE REVIEW

File: src/yang/ietf-tcp.yang [hunk 1]
```diff
@@ -170,25 +171,27 @@ module ietf-tcp {
         leaf local-address {
           type inet:ip-address;
           description
-            "Local address that forms the connection identifier.";
+            "Identifies the address that is used by the local
+             endpoint for the connection.";
```
Comment (mahesh-arrcus) [inline src/yang/ietf-tcp.yang]: The local-address is part of the key (connection identifier). Therefore, I would say:

"Identifies the address that is used by the local endpoint for the connection, and is one of the four elements that form the connection identifier."

File: src/yang/ietf-tcp.yang [hunk 2]
```diff
@@ -170,25 +171,27 @@ module ietf-tcp {
...
           description
-            "Local TCP port that forms the connection identifier.";
+            "Local TCP port used for the connection.";
         }
 
         leaf remote-port {
           type inet:port-number;
           description
-            "Remote TCP port that forms the connection identifier.";
+            "Remote TCP port used for the connection.";
```
Comment (mahesh-arrcus) [inline src/yang/ietf-tcp.yang]: How about this?

"Identifies the remote TCP port used for the connection, and is one of the four elements that form the connection identifier."

File: src/yang/ietf-tcp.yang [hunk 3]
```diff
@@ -82,7 +82,8 @@ module ietf-tcp {
       type boolean;
       default "false";
       description
-        "Enable support of TCP-Authentication Option (TCP-AO).";
+        "Enable support of TCP-Authentication Option (TCP-AO) when set
+         to true.";
```
Comment (mahesh-arrcus) [inline src/yang/ietf-tcp.yang]: Would it not be better to say:

"When set to true, TCP-Authentication Option (TCP-AO) is enabled."

File: src/yang/ietf-tcp.yang [hunk 4]
```diff
@@ -116,7 +117,7 @@ module ietf-tcp {
       must "../enable-ao = 'true'";
       default true;
       description
-        "Include TCP options in MAC calculation.";
+        "Include TCP options in MAC calculation when set to true.";
```
Comment (mahesh-arrcus) [inline src/yang/ietf-tcp.yang]: Similarly,

"When set to true, TCP options are included in MAC calculation."

File: src/yang/ietf-tcp.yang (hunk 1 above)
Comment (vmurgai) [reply src/yang/ietf-tcp.yang]: looks good to me.

File: src/yang/ietf-tcp.yang (hunk 2 above)
Comment (vmurgai) [reply src/yang/ietf-tcp.yang]: looks good to me.

File: src/yang/ietf-tcp.yang (hunk 3 above)
Comment (vmurgai) [reply src/yang/ietf-tcp.yang]: looks good to me.

File: src/yang/ietf-tcp.yang (hunk 4 above)
Comment (vmurgai) [reply src/yang/ietf-tcp.yang]: looks good to me.

Comment (michael-scharf) [reply src/yang/ietf-tcp.yang]: Fixed

File: src/yang/ietf-tcp.yang (hunk 3 above)
Comment (michael-scharf) [reply src/yang/ietf-tcp.yang]: Fixed

File: src/yang/ietf-tcp.yang (hunk 2 above)
Comment (michael-scharf) [reply src/yang/ietf-tcp.yang]: Fixed

File: src/yang/ietf-tcp.yang (hunk 1 above)
Comment (michael-scharf) [reply src/yang/ietf-tcp.yang]: Fixed

**Review by mjethanandani — APPROVED** [review]

File: src/yang/ietf-tcp.yang [hunk 5]
```diff
@@ -82,7 +82,8 @@ module ietf-tcp {
       type boolean;
       default "false";
       description
-        "Enable support of TCP-Authentication Option (TCP-AO).";
+        "When set to true, TCP-Authentication Option (TCP-AO) is
+         enabled."
```
Comment (mjethanandani) [inline src/yang/ietf-tcp.yang]: You are still missing the ; at the end of this statement, which is leading to this error:

Validating yang/ietf-tcp@2022-01-18.yang
yang/ietf-tcp@2022-01-18.yang:87: error: unterminated statement definition for keyword "description", looking at }
yang/ietf-tcp@2022-01-18.yang failed pyang validation

Comment (michael-scharf) [reply src/yang/ietf-tcp.yang]: Sorry, I missed that one. It should now be fixed.

**Review by mjethanandani — APPROVED** [review]

**Pull request merged**